okhttp/src/main/java/okhttp3/internal/cache/CacheInterceptor.java

```diff
@@ -256,4 +254,14 @@ static boolean isEndToEnd(String fieldName) {
         && !"Transfer-Encoding".equalsIgnoreCase(fieldName)
         && !"Upgrade".equalsIgnoreCase(fieldName);
   }
+
+  /**
+   * Returns true if {@code fieldName} should be ignored, when combining cache headers with
+   * network headers.
+   */
+  static boolean isNotContentSpecificHeader(String fieldName) {
```

Comment:
nit: Could we negate this? it's used both ways but I don't like no double negatives